misc/extension_api_validation/4.2-stable.expected

```diff
@@ -14,3 +14,10 @@ Validate extension JSON: Error: Field 'classes/TileData/methods/get_navigation_p
 Validate extension JSON: Error: Field 'classes/TileData/methods/get_occluder/arguments': size changed value in new API, from 1 to 4.
 
 Added optional argument to get_navigation_polygon and get_occluder to specify a polygon transform.
+
+
+GH-86661
+--------
+Validate extension JSON: Error: Field 'classes/Animation/methods/track_find_key/arguments': size changed value in new API, from 3 to 4.
+
+Added optional argument to track_find_key to avoid finding keys out of the animation range. 
```

Comment:
```suggestion
Added optional argument to track_find_key to avoid finding keys out of the animation range. Compatibility method registered.
```
I'd add this note (also trailing space)